db/migrate/20170420160820_copy_description_values_to_description_string.rb

```diff
@@ -0,0 +1,12 @@
+class CopyDescriptionValuesToDescriptionString < ActiveRecord::Migration[5.0]
+  def up
+    Edition.all.each do |edition|
```

Comment:
you'll probably find this is a bit quicker if you use `find_each` as that'll pull out batches of 1000.